src/main/java/org/apache/commons/csv/CSVRecord.java

```diff
@@ -48,6 +48,11 @@ public final class CSVRecord implements Serializable, Iterable<String> {
      */
     private final long characterPosition;
 
+    /**
+     * The start byte of this record as a character byte in the source stream.
```

Comment:
This comment is confusing because it talks about a `byte` but the type is a `long`. It would be better to explain the mismatch, if that is indeed the intent.

Reply:
We are calculating the number of bytes required to encode a single character and storing this value as a `long`. This choice provides a larger range for the position and ensures consistency with the design, as position is also defined as a `long`.